README.md

```diff
@@ -65,6 +65,19 @@ In order to use ArcGIS maps, you must [register](https://developers.arcgis.com/e
 
 [Available Esri layers](http://leaflet-extras.github.io/leaflet-providers/preview/#filter=Esri)
 
+### TomTom
+
+In order to use TomTom layers, you must [register](https://developer.tomtom.com/user/register). Once registered, you can create an `apikey` which you have to pass to `L.tileLayer.provider` in the options:
+
+```Javascript
+L.tileLayer.provider('TomTom', {
+    apikey: '<insert your API key here>'
+}).addTo(map);
+```
+
+[Available HERE layers](http://leaflet-extras.github.io/leaflet-providers/preview/#filter=HERE)
```

Comment:
Should be a link related to tomtom layers instead of HERE ;)

Reply:
Done!